setup_data_scheme.md

```diff
@@ -64,19 +64,24 @@ name | Type | Description
 ------- | ------- | --------------
 *type* | String | Type of association, either `one_to_one`, `one_to_many`, `many_to_one`, or `many_to_many`.
 *target* | String | Name of model to which the current model will be associated with.
-*implementation* | String | implementation type of the association. Can be one of `foreignkey`, `generic` or `sql_cross_table` (only for `many_to_many`)`
+*implementation* | String | implementation type of the association. Can be one of `foreignkey`, `generic` or `sql_cross_table` (only for `many_to_many`)
 *reverseAssociation* | String | The name of the reverse association from the other model. This field is only mandatory for building the [single-page-app](https://github.com/Zendro-dev/single-page-app), *not* for generating the the graphql-server code via this repository.
-*targetKey* | String | A unique identifier of the association stored in any of the two models involved in the association.
-*keysIn* | String | Name of the model where the targetKey is stored.
 *targetStorageType* | String | Type of storage where the target model is stored.
-*label* | String | Name of the column in the target model to be used as a display name in the GUI. This will be useful as a preview of the association.
-*sublabel* | String | Optional name of the column in the target model to be used as a sub-label in the GUI. Also used for a more explicit preview of the association.
 *useDataLoader* | Boolean | If it is set to `true`, server could fetch multiple records within one query for `readOne<model>` API. 
 
-**Note**: The `keysIn` argument points to the model that stores the information about the foreignKey(s). That can be either a single key, a foreignkey array or a cross-model.
 
 ### Foreign keys
 It's important to notice that when a model involves a foreign key for the association, this key should be explicitly written into the attributes field of the given local model. Although, foreign keys will be available for the user only as readable attributes, for editing this attributes we offer the possibility as part of the API, please see [this](api_graphql.md#extra-mutation-fields-to-update-or-create-associations) section for more info.
+To store to-many associations (many-to-many or one-to-many) via foreign keys Zendro offers to store the foreign keys in arrays. In this case the model will have an array attribute which will store ids from the associated records.
+
+There are two ways for storing foreign keys in Zendro, namely `single-end foreign keys` and `paired-end foreign keys`. For one association with two data models, single-end foreign keys would only be stored in one data model, while paired-end foreign keys would be saved in both data models.
```

Comment:
>For one association with two data models, single-end foreign keys would only be stored in one data model, while paired-end foreign keys would be saved in both data models.
For a association between two data models, single-end foreign keys will only be stored in a single data model, while paired-end foreign keys will be saved in both data models.